README.md

```diff
@@ -42,140 +42,22 @@ MuBench CI Status: [![MuBench CI Status](https://api.shippable.com/projects/570d
-2. "Mine and Detect"-mode, where the detector should mine its own patterns in the provided code base and find violations in that same code base. Again, input is provided as source code and corresponding Bytecode.
-
-The `DetectorOutput` is essentially a collection where you add your detector's findings. MUBench expects you to add the findings ordered by the detector's confidence, descending.
-
-To register your own detector to MUBench, the following steps are necessary:
-
-1. Create a new subfolder `my-detector` in the [detectors](https://github.com/stg-tud/MUBench/tree/master/detectors) folder. `my-detector` will be the Id used to refer to your detector when running experiments.
-2. Add the executable JAR with your detector as `my-detector/my-detector.jar`.
-3. Run MUBench as usual.
-
-### Run on Your Project
-
-MUBench is designed to run detectors on the benchmark projects that come with the it. Nevertheless, you can also use MUBench to run a detector on your own code, with a few simple steps:
-
-1. Create the folder `data/<project>/versions/<version>/compile`, with arbitrary names for `project` and `version`.
-2. Copy/move your project code into that `compile` folder.
-3. Create a file `data/<project>/project.yml` with the content:
-    ```
-    name: <Your Project's Display Name>
-    repository:
-      type: synthetic
-    ```
-    This instructs MUBench to use the `compile` folder as the project's "repository". Note that MUBench will copy the entire folder in its compile phase.
-    
-4. Create a file `data/<project>/versions/<version>/version.yml` with the content:
-    ```
-    build:
-      src: "<src-root>"
-      commands:
-        - echo "fake build"
-      classes: "<classes-root>"
-    misuses: []
-    revision: 0
-    ```
-    
-    The values for `src-root`/`classes-root` are the relative paths to the source/classes folders within the `compile` folder. If you pre-build your project and, thus, have the classes already available, you have to use the fake-build command above to satisfy MUBench. If you need to execute any commands in order to build the project, you may list these below the `commands` key. Our Docker container comes with a couple of build tools, such as Maven, Gradle, and Ant, but may not satisfy all your build requirements. If you only want to run detectors that work on source code, such as MuDetect, you can stay with the fake build, too.
-5. Run a detector, e.g., MuDetect, with `./mubench detect MuDetect 2 --only <project>.<version>`.
-6. To upload the results to a review site, run `./mubench publish findings MuDetect 2 --only <project>.<version> -s http://<your-sites.url>/index.php/ -u <username> -p <password>` (this will also run detection, if necessary).
+* [Pipeline](mubench.pipeline/)
+* [Review Site](mubench.reviewsite/)
+* [Detectors](mubench.cli/)
+* [Misuses](mubench.pipeline/)
```

Comment:
"Add Your Own Project or Misuse Examples".

Please split the respective information into a separate README in data/.